src/main/resources/templates/mdBenefitsFlow/howThisWorks.html

```diff
@@ -7,91 +7,69 @@
+          <th:block th:ref="after">
+            <p th:text="#{how-this-works.after-l1}"></p>
+            <p th:text="#{how-this-works.after-l2}"></p>
           </th:block>
         </th:block>
-
-        <th:block th:replace="~{'fragments/honeycrisp/reveal' :: reveal(
-                controlId='r3',
-                linkLabel=~{::revealLabel3},
-                content=~{::revealContent3})}">
-          <th:block th:ref="revealLabel3">
-            <i class="icon-">&#xE862;</i>
-            <span th:text="#{how-this-works.adding-documents}"></span>
-          </th:block>
-          <th:block th:ref="revealContent3">
-            <ul class="list--bulleted">
-              <li th:text="#{how-this-works.adding-documents-l1}"></li>
-              <li th:text="#{how-this-works.adding-documents-l2}"></li>
-              <li th:text="#{how-this-works.adding-documents-l3}"></li>
-            </ul>
+        <th:block th:replace="~{fragments/honeycrisp/accordion :: accordion(
+          buttonLabel=#{how-this-works.how-soon},
+          content=~{::how-soon})}">
+          <th:block th:ref="how-soon">
+            <p th:text="#{how-this-works.how-soon-l1}"></p>
+            <p th:text="#{how-this-works.how-soon-l2}"></p>
+            <p th:text="#{how-this-works.how-soon-l3}"></p>
           </th:block>
         </th:block>
-
-        <th:block th:replace="~{'fragments/honeycrisp/reveal' :: reveal(
-                controlId='r4',
-                linkLabel=~{::revealLabel4},
-                content=~{::revealContent4},
-                forceShowContent='true')}">
-          <th:block th:ref="revealLabel4">
-            <i class="icon-">&#xE862;</i>
-            <span th:text="#{how-this-works.after-you-submit}"></span>
+        <th:block th:replace="~{fragments/honeycrisp/accordion :: accordion(
+          buttonLabel=#{how-this-works.minimum-application},
+          content=~{::minimum-application})}">
+          <th:block th:ref="minimum-application">
+            <p th:text="#{how-this-works.minimum-application-l1}"></p>
+            <p th:utext="#{how-this-works.minimum-application-l2}"></p>
+            <p th:utext="#{how-this-works.minimum-application-l3}"></p>
+            <p th:utext="#{how-this-works.minimum-application-l4}"></p>
           </th:block>
-          <th:block th:ref="revealContent4">
-            <p th:text="#{how-this-works.after-you-submit-l1}"></p>
-            <p th:utext="#{how-this-works.after-you-submit-l2}"></p>
-            <p th:text="#{how-this-works.after-you-submit-l3}"></p>
+        </th:block>
+        <th:block th:replace="~{fragments/honeycrisp/accordion :: accordion(
+          buttonLabel=#{how-this-works.add-documents},
+          content=~{::add-documents})}">
+          <th:block th:ref="add-documents"
+                    th:with="email=${inputData.county == 'BALTIMORE' ? '<a href=&quot;mailto:baltimorecounty.dmc@maryland.gov&quot; target=&quot;_blank&quot;>baltimorecounty.dmc@maryland.gov</a>' : 
```

Comment:
Can you use the Counties enum here?  Thanks

Reply:
I debated requesting this, but if we change the wording, we will then have to change it in multiple spots, if we have hard coding like this.